lizmap/modules/admin/controllers/landing_page_content.classic.php

```diff
@@ -0,0 +1,107 @@
+<?php
+/**
+ * Lizmap administration : landing page content.
+ *
+ * @author    3liz
+ * @copyright 2016 3liz
+ *
+ * @see      http://3liz.com
+ *
+ * @license Mozilla Public License : http://www.mozilla.org/MPL/
+ */
+class landing_page_contentCtrl extends jController
+{
+    // Configure access via jacl2 rights management
+    public $pluginParams = array(
+        '*' => array('jacl2.right' => 'lizmap.admin.access'),
+    );
+
+    /**
+     * Display a wysiwyg editor.
+     */
+    public function index()
+    {
+        $rep = $this->getResponse('html');
+
+        // Get HTML content from ini file
+        $lizmapConfigIniFile = jApp::configPath('lizmapConfig.ini.php');
+        $iniModifier = new jIniFileModifier ($lizmapConfigIniFile);
+
+        $HTMLContent = html_entity_decode($iniModifier->getValue("html", "landing_page_content"));
+
+        // Create the form
+        $form = jForms::create('admin~landing_page_content');
+        $form->setData( "HTMLContent", $HTMLContent );
+
+        $tpl = new jTpl();
+
+        $tpl->assign('form', $form);
+        $rep->body->assign('MAIN', $tpl->fetch('landing_page_content'));
+        $rep->body->assign('selectedMenuItem', 'lizmap_landing_page_content');
+
+        return $rep;
+    }
+
+    /**
+     * Save wysiwyg editor content in ini file.
+     */
+    public function save()
+    {
+        $form = jForms::get('admin~landing_page_content');
+
+        // token
+        $token = $this->param('__JFORMS_TOKEN__');
+        if (!$token) {
+            // redirection vers la page d'erreur
+            $rep = $this->getResponse('redirect');
+            $rep->action = 'landing_page_content:index';
+
+            return $rep;
+        }
+
+        // If the form is not defined, redirection
+        if (!$form) {
+            $rep = $this->getResponse('redirect');
+            $rep->action = 'landing_page_content:index';
+
+            return $rep;
+        }
+
+        // Set the other form data from the request data
+        $form->initFromRequest();
+
+        // Check the form
+        $ok = true;
+        if (!$form->check()) {
+            $ok = false;
+        }
+
+        if (!$ok) {
+            // Errors : redirection
+            $rep = $this->getResponse('redirect');
+            $rep->action = 'landing_page_content:index';
+            $rep->params['errors'] = '1';
+
+            return $rep;
+        }
+
+        // Remove line breaks, spaces between > and < then transform with htmlentities to get a single string
+        $HTMLContent = htmlentities(preg_replace('/>\s+</','><',str_replace(array("\r", "\n"), '', $form->getData('HTMLContent'))));
+
+        // Save the data
+        $lizmapConfigIniFile = jApp::configPath('lizmapConfig.ini.php');
+        $iniModifier = new jIniFileModifier ($lizmapConfigIniFile);
+
+        $iniModifier->setValue("html", $HTMLContent, "landing_page_content");
```

Comment:
I'm not sure lizmapConfig.ini.php is the right place to store some HTML content...